dictionaries/en_GB/src/exclude-words.txt

```diff
@@ -1,2 +1,4 @@
+~colum
+colum
 knifes
 midwifes
```

Comment:
why this while you added them 4 to dictionaries/en_shared/src/exclude-words.txt

Reply:
Good catch, I though I had deleted this copy.